### PR TITLE
Migration script for resultBlocks field

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?
This PR adds a Contentful Migration script to create a new `resultBlocks` field for the `Quiz` Content Type.

### Any background context you want to provide?
This is probably the most crucial field that we need to formalize since as things are now, one would need to manually paste in all the necessary fields for the result blocks into Additional Content 😨 

### What are the relevant tickets/cards?
Refs #780 
[Pivotal #155783121](https://www.pivotaltracker.com/story/show/155783121)

image

### CODE REVIEW

**Comment by mendelB** [comment]
One thing I wasn't entirely sure about with this one was should the field be required?

We do have a [fallback in place](https://github.com/DoSomething/phoenix-next/blob/master/resources/assets/components/Quiz/Quiz.js#L70) for if no `resultBlock` is found for the Quiz result, but do we want to encourage actual Blocks as quiz results, or either way is OK?

**Comment by weerd** [comment]
I don't think it should be required, but maybe it's a question for @ashleybaldwin?

File: contentful/migrations/2018_03_29_001_add_result_blocks_field_to_quiz.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = function (migration) {
+  const quiz = migration.editContentType('quiz');
+
+  quiz.createField('resultBlocks')
+    .name('Result Blocks')
+    .type('Link')
+    .linkType('Entry')
```
Comment (weerd) [inline contentful/migrations/2018_03_29_001_add_result_blocks_field_to_quiz.js]: Is this only a single reference and not a many reference? The old quiz beta results field was a many reference, so you could include both alternatives depending on outcome of quiz... either a link to external site if over 18 or a share action if under 18. Does this work differently now if it's a single reference?

Comment (mendelB) [reply contentful/migrations/2018_03_29_001_add_result_blocks_field_to_quiz.js]: omg no this should be multi reference 🤦‍♂️ 
I made this last night after too much champaign it seems. I'll update!

Comment (mendelB) [reply contentful/migrations/2018_03_29_001_add_result_blocks_field_to_quiz.js]: OK, updated! (Yeesh, good catch!)

**Comment by mendelB** [comment]
Confirmed with @ashleybaldwin, optional field it is!

**Pull request merged**